shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/mapper/OnUpdateFeedingPointToDomainMapper.kt

```diff
@@ -23,5 +23,8 @@ internal fun OnUpdateFeedingPoint.toDomainFeedingPoint() =
         location = MapLocation(
             latitude = location().lat(),
             longitude = location().lon()
-        )
+        ),
+        images = with(images()) {
```

Comment:
Maybe it simplyfied?
```suggestion
        images() ?: emptyList()
```

Reply:
Looks better, thanks!